torch/onnx/symbolic_opset13.py

```diff
@@ -137,14 +137,27 @@ def fake_quantize_per_channel_affine(g, inputs, scale, zero_point, axis, quant_m
 
     # ONNX defines zero_point to be int8 or uint8
     if quant_min == 0:
-        zero_point = g.op("Cast", zero_point, to_i=sym_help.cast_pytorch_to_onnx["Byte"])
+        zero_point = g.op("Cast", zero_point, to_i=torch.onnx.TensorProtoDataType.UINT8)
     else:
-        zero_point = g.op("Cast", zero_point, to_i=sym_help.cast_pytorch_to_onnx["Char"])
+        zero_point = g.op("Cast", zero_point, to_i=torch.onnx.TensorProtoDataType.INT8)
     return g.op(
         "DequantizeLinear",
         g.op("QuantizeLinear", inputs, scale, zero_point, axis_i=axis),
         scale, zero_point, axis_i=axis)
 
+@parse_args("v", "v", "v", "i", "i")
+def fake_quantize_per_tensor_affine(g, inputs, scale, zero_point, quant_min=-128, quant_max=127):
+    if quant_min not in [0, -128] or quant_max not in [127, 255]:
```

Comment:
Should the check be:
`if (quant_min, quant_max) not in ((-128, 127), (0, 255))):`
?
Otherwise you can have  `quant_min = -128` and `quant_max = 255`, which I guess should not be allowed.

If you change it here please change it in symbolic_opset10.py as well.

Reply:
Good catch, I believe the previous condition is not precise enough. Updated original code as well.